src/Translatable/Translatable.php

```diff
@@ -111,7 +111,17 @@ public function getTranslationModelName()
      */
     public function getTranslationModelNameDefault()
     {
-        return get_class($this).config('translatable.translation_suffix', 'Translation');
+        $fullModelName = $this->getTranslationModelNamespace().'\\'.class_basename(get_class($this));
+
+        return $fullModelName.config('translatable.translation_suffix', 'Translation');
+    }
+
+    /**
+     * @return string
+     */
+    public function getTranslationModelNamespace()
+    {
+        return config('translatable.translation_model_namespace', 'App');
```

Comment:
This changes the default logic and is BC.
I won't merge this feature if it's BC because it's too specific.

To solve this the default config value should be `null` als fallback it should use the namespace of `get_class()`.

Atm it will also work, by default, if model namespace is moved into `App\Models` or anywhere else. After this change I have to adjust the config to keep this running. That's why I want to keep the default behavior and just add the option to adjust the namespace.